### ISSUE TITLE
Resource Type as URL and ID

### ISSUE THREAD

**Issue by smizell** [alps-io/spec#79]
I've seen `rt` [used as a URL](http://alps.io/schema.org/Thing.xml), but I see [in the docs](http://tools.ietf.org/html/draft-amundsen-richardson-foster-alps-01#section-2.2.11) it may match the `id` of another descriptor. Should the spec provide clarity on this? Is this meant to allow the user to use it how they wish?

If it's desired to make this specific, one option I see is to make it a resolvable URL like `href`, which means you can reference other descriptors in the same document with a fragment just like you do with `href`.

This is what the example from the docs would look like:

```xml
<alps version="1.0">
  <doc format="text">A contact list.</doc>

  <!-- a hypermedia control for returning contacts -->
  <descriptor id="collection" type="safe" rt="#contact">
    <doc>
      A simple link/form for getting a list of contacts.
    </doc>
    <descriptor id="nameSearch" type="semantic">
      <doc>Input for a search form.</doc>
    </descriptor>
  </descriptor>

  <!--  a contact: one or more of these may be returned -->
  <descriptor id="contact" type="semantic">
    <descriptor id="item" type="safe">
      <doc>A link to an individual contact.</doc>
    </descriptor>
    <descriptor id="fullName" type="semantic" />
    <descriptor id="email"    type="semantic" />
    <descriptor id="phone"    type="semantic" />
  </descriptor>
</alps>
```

**Comment by mamund** [comment]
yep - this is a spec bug, i think. the value of RT SHOULD be a valid URL
pointer to an existing descriptor ID. for in-doc pointers, you MAY use hash
values ("#contact") for external pointers you MUST use either a relative
URL ("/profiles/users#contact") or absolute URLs ("
http://api.example.org/profiles/users#contact").

happy to discuss further, but if we're all cool w/ this, feel free to work
up a PR for this one.
​

**Comment by filip26** [comment]
Is there any reason to use URL and not URI/IRI? I would prefer:

  * 'rt' value MUST be valid URI

It solves the issue, and also allows to use 'well-know' IRI as a definitive identifier.

btw. what about adding a label `errata`? it would be easier to search later.
